tests/base_tests.py

```diff
@@ -44,6 +44,26 @@
 FAKE_DB_NAME = "fake_db_100"
 
 
+def login(client, username="admin", password="general"):
```

Comment:
```suggestion
def login(client: Any, username: str = "admin", password: str = "general"):
```
(AFAIK there isn't a defined type for `TestCase.client`)